utils/utils.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"math/rand"
+	"strings"
+
+	client "github.com/fybrik/datacatalog-go-client"
+	models "github.com/fybrik/datacatalog-go-models"
+)
+
+func AppendStrings(a string, b string) string {
+	if strings.Contains(b, ".") {
+		return a + ".\"" + b + "\""
+	} else {
+		return a + "." + b
+	}
+}
+
+// If the tag is in the Fybrik tag category (e.g. "Fybrik.PII"), remove the
+// "Fybrik." prefix. Otherwise, do nothing
+func StripTag(tag string) string {
+	return strings.TrimPrefix(tag, "Fybrik.")
+}
+
+func UpdateCustomProperty(customProperties map[string]interface{}, orig map[string]interface{}, key string, value *string) {
+	// update customProperties only if there is a new value
+	if value != nil && *value != "" {
+		customProperties[key] = *value
```

Comment:
why do you need pointers to strings ? and why do you path value as a pointer to string ?

Reply:
in the auto-generated code, most properties (geography, owner, dataFormat) are represented as pointers to string.
I prefer pointers to string, because, unlike string, they can be assigned a nil value.